package.json

```diff
@@ -42,13 +43,17 @@
     "react": "17.0.2",
     "react-dom": "17.0.2",
     "react-i18next": "11.18.6",
+    "react-json-tree": "^0.19.0",
     "react-outside-click-handler": "^1.3.0",
     "react-resize-detector": "7.1.2",
     "react-router": "5.3.4",
     "react-router-dom": "5.3.4",
     "react-scripts": "5.0.1",
     "sass": "1.77.1",
-    "styled-components": "5.3.11"
+    "string-width": "4.2.3",
+    "strip-ansi": "6.0.1",
+    "styled-components": "5.3.11",
+    "wrap-ansi": "7.0.0"
```

Comment:
These 3 new dependents are not used in the codebase

Reply:
[Here](https://github.com/ckb-js/lumos/blob/c41f602d1ce47c8ee656c871ed960979619c694b/examples/molecule-codegen/package.json#L7) is an example of `lumos/molecule` without explicitly installing these packages, and it works. Could you please share the error message when dropping these dependencies?

Reply:
> [Here](https://github.com/ckb-js/lumos/blob/c41f602d1ce47c8ee656c871ed960979619c694b/examples/molecule-codegen/package.json#L7) is an example of `lumos/molecule` without explicitly installing these packages, and it works. Could you please share the error message when dropping these dependencies?

I deleted it and tried to build it and found it was fine... I'm sorry for wasting your time.